easier68k/simulator/m68k.py

```diff
@@ -275,3 +275,37 @@ def save_memory(self, file : typing.BinaryIO):
         NOTE: file must be opened as binary or this won't work
         """
         self.memory.save_memory(file)
+
+
+    def set_ccr_reg(self, extend, negative, zero, overflow, carry): 
+        """
+        Accepts Boolean values for X,N,Z,V, and C, respectively and sets the CCR accordingly.
+        Passing None in for any argument will cause it to ignore that bit.
+        Returns nothing.
+        :param extend:
+        :param negative:
+        :param zero:
+        :param overflow:
+        :param carry:
+        :return:
+        """
+        if extend != None:
```

Comment:
```suggestion
        if extend is not None:
```
Please refer to: https://stackoverflow.com/a/3965129